readme.adoc

```diff
@@ -13,56 +22,40 @@ project. Sorry.
 === Intended reader
 
 The typical reader is a developer who just started digging into
-Bitcoin development or have worked in the Bitcoin space in one form or
-the other for a while.
+Bitcoin development or has worked in the Bitcoin space in one form or
+the other.
 
-The reader doesn't have to know every ins an outs of Bitcoin, but
+The reader doesn't have to know all the ins and outs of Bitcoin, but
 should understand how it operates on a high level. For example, the
 following should be clear to the reader beforehand:
 
 * What proof-of-work is
 * How blocks are interconnected
-* That transactions are usually signed using private keys before being
-  published to the Bitcoin peer-to-peer network.
-
-The reader should also be humble and willing to re-evaluate his/her
-opinions. The materials we present here are written by people who have
-been studying Bitcoin for a very long time and some were into digital
-money way before Bitcoin existed (yes, they existed before Bitcoin).
+* Lifecycle of transactions
 
 === How is this organized?
 
 The project is sectioned into topics. Each topic will guide you
 through a number of links to articles or videos, that we recommend
-reading and we briefly discuss each link.
+reading and briefly discuss each link.
```

Comment:
Add "The material presented is written by individuals who have studied Bitcoin for a long time. "